<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QDialog" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>590</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="group_TestOptions">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>561</width>
     <height>571</height>
    </rect>
   </property>
   <property name="title">
    <string>Test Options</string>
   </property>
   <widget class="QCheckBox" name="cbox_CalibrateXCSEL">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>151</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Calibrate XCSEL</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbox_DTMTest">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>151</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>DTM Test</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="group_DTM">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>541</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>DTM (Bluetooth Low Energy RF PHY Test Suite)</string>
    </property>
    <widget class="QGroupBox" name="group_Transmitter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>521</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Transmitter</string>
     </property>
     <widget class="QPushButton" name="btn_OutputPower">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>31</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbox_OutputPower">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>18</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>TRM-LE/CA/01/C(Output Power)</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_ModulationCharacteristics">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>31</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_CarrierFrequencyOffset">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>31</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbox_ModulationCharacteristics">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>TRM-LE/CA/05/C(Modulation characteristics)</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbox_CarrierFrequencyOffsetAndDrift">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>431</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>TRM-LE/CA/01/C(Carrier frequency offset and Drift)</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="group_Receiver">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>521</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Receiver</string>
     </property>
     <widget class="QPushButton" name="btn_ReceiverSensitivity">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>31</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbox_ReceiverSensitivity">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>18</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>RCV-LE/CA/01/C(Receiver Sensitivity)</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_MaximumInputSignalLevel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>31</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_PERReportIntergrity">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>31</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbox_MaximumInputSignalLevel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>RCV-LE/CA/06/C(Maximum Input signal level)</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbox_PERReportIntergrity">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>RCV-LE/CA/07/C(PER Report Intergrity)</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group_DownloadOptions">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>541</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Download Options</string>
    </property>
    <widget class="QGroupBox" name="group_Increase">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>291</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>Increase</string>
     </property>
     <widget class="QGroupBox" name="group_CompanyID">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Company ID</string>
      </property>
      <widget class="QLineEdit" name="le_CompanyID0Bit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>08</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_CompanyID1Bit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>7C</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_CompanyID2Bit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>BE</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btn_Set">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>93</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="group_StartPart">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Start Part</string>
      </property>
      <widget class="QLineEdit" name="le_StartPart0Bit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>00</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_StartPart1Bit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>00</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_StartPart2Bit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>00</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="group_EndPart">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>End Part</string>
      </property>
      <widget class="QLineEdit" name="le_EndPart0Bit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>FF</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_EndPart1Bit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>FF</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_EndPart2Bit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>FF</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QGroupBox" name="group_BDName">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>BD Name</string>
     </property>
     <widget class="QCheckBox" name="cbox_NameAddress">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>181</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Name + Address</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="group_BurnOptions">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>70</y>
       <width>221</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>Burn Options</string>
     </property>
     <widget class="QCheckBox" name="cbox_BurnApplication">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>181</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Burn Application</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbox_BurnOTA">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>181</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Burn OTA</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbox_BurnData">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>181</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Burn Data</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbox_BurnNVDS">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>181</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Burn NVDS</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QRadioButton" name="rbtn_Default">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>115</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Default</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Setting</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Setting</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
